File: SampleSetup/Protocol/Body/ApiTokenPostReturn.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace SampleProject.Protocol.Body
{
    public class ApiTokenPostReturn
    {
        public string access_token = "";
        public string ExpireDateIso = "";
        public Dictionary<string, string> TestDict { get; set; } = new Dictionary<string, string>();
    }
}

File: SampleSetup/Output/PHP_8/Body/ApiTokenPostReturn.php
<?php
namespace SampleProject\Protocol\Body;

class ApiTokenPostReturn {
    public string $access_token;
    public string $ExpireDateIso;
    /**
     * @var array
     */
    public $TestDict;

    public function __construct()
    {
        $this->access_token = "";
        $this->ExpireDateIso = "";
        $this->TestDict = [];
    }
}
